### ISSUE TITLE
Provide Examples (Single-Files)

### ISSUE THREAD

**Issue by Petschko** [Petschko/dhl-php-sdk#8]
The `test.php` File is a mess and I want make more examples instead of this one with many comments etc

**Comment by EddyFK** [comment]
I have a pretty hard time to set up international shipping, the example from readme gives a lot of validation errors... is that version still up to date?

**Comment by Petschko** [comment]
Yes it is up to date... But I will make them soon if I get a bit time for that.

May I can help you until then with personal support? You can send me a Mail with your problems... (At least until I made the examples done)

peter@petschko.org

Sorry for the late answer, I was not at home for 1 Week =/
